test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -4283,24 +4271,34 @@ def create_pbsnode(self, node_name, attrs):
                             'bin', 'qmgr')
         execcmd = "create node " + node_name
         cmd = [qmgr, "-c", execcmd]
-        self.du.run_cmd(
-            self.hostname, cmd, sudo=True)
-        skip_atb_list = ['id', 'Port', 'pbs_version', 'pcpus',
-                         'state', 'last_state_change_time', 'ntype',
+        ret = self.du.run_cmd(self.hostname, cmd, sudo=True)
+        if ret['rc'] != 0:
+            self.logger.info("Failed to create node: %s" % node_name)
+            self.logger.error("Error: %s" % ret['err'])
```

Comment:
Since you didn't provide logerr=False to run_cmd, it will print out ret['err'] automatically, so please remove this line.

Reply:
changed